src/main/resources/templates/categories_list.html

```diff
@@ -8,41 +8,50 @@
     <title>Document</title>
 </head>
 <body>
-    <h2>Categories</h2>
-    <a href="/categories/add">Add Category</a>
-    <table border="1" cellpadding="5" cellspacing="2">
-        <thead>
-            <tr>
-                <th>Id</th>
-                <th>Name</th>
-                <th>Slug</th>
-                <th>Actions</th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr th:each="category : ${categories}">
-                <td th:text="${category.id()}"></td>
-                <td th:text="${category.name()}"></td>
-                <td th:text="${category.slug()}"></td>
-                <td>
-                    <button>🖋</button>
-                    <button>🗑️</button>
-                </td>
-            </tr>
-        </tbody>
-    </table>
-    <div th:if="${success}" id="successDiv">Category inserted successfully</div>
-    <div th:if="${failure}" id="failureDiv">Insertion failure</div>
+<h2>Categories</h2>
+<a href="/categories/add">Add Category</a>
+<table border="1" cellpadding="5" cellspacing="2">
+    <thead>
+    <tr>
+        <th>Id</th>
+        <th>Name</th>
+        <th>Slug</th>
+        <th>Actions</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr th:each="category : ${categories}">
+        <td th:text="${category.id()}"></td>
+        <td th:text="${category.name()}"></td>
+        <td th:text="${category.slug()}"></td>
+        <td>
+            <button>🖋</button>
+            <a th:href="@{/categories/delete(id=${category.id()})}">🗑️</a>
+        </td>
+    </tr>
+    </tbody>
+</table>
+<div th:if="${success}" id="successDiv">Category inserted successfully</div>
+<div th:if="${failure}" id="failureDiv">Insertion failure</div>
 
-    <script>
-        const successDiv = document.querySelector("#successDiv");
-        const failureDiv = document.querySelector("#failureDiv");
+<script>
+    const successDiv = document.querySelector("#successDiv");
+    const failureDiv = document.querySelector("#failureDiv");
+    const urlParams = new URLSearchParams(window.location.search);
+    const categoryDeleted = urlParams.get('categoryDeleted');
+    const deletionFailed = urlParams.get('deletionFailed');
 
-        setTimeout(() => {
-            successDiv.style.display = "none";
-            failureDiv.style.display = "none";
-        }, 3000);
+    if (categoryDeleted) {
```

Comment:
Let's be consistent and add the deleted message the same way as the other messages.
Either all message boxes or all alerts... but message boxes are better :D